connector/oidc/oidc.go

```diff
@@ -425,6 +431,28 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 				}
 			}
 		}
+		c.logger.Debugf("Groups::: %q \n", groups)
+
+		// Validate that the user is part of allowedGroups
+		var accessAllowed = false
+		if len(c.allowedGroups) > 0 {
+			for _,group := range c.allowedGroups {
+				groupPresent := slices.Contains(groups, group)
+				if groupPresent {
+					accessAllowed = true
+					c.logger.Debugf("Found a match - Group::: %q \n", group)
+					break
+				}
+			}
+		} else {
+			// don't check for groups if allowedGroups is not configured
+			c.logger.Debugf("Group check not performed \n")
```

Comment:
this looks unnecessary

Reply:
Will remove it.

Reply:
Handled in 14f867af2fa25cd4a440c7779431223ab55796af